pom.xml

```diff
@@ -54,7 +54,7 @@
     <testng.version>6.8</testng.version>
     <guava.version>17.0</guava.version>
     <ome-common.version>5.3.1</ome-common.version>
-    <ome-model.version>5.3.1</ome-model.version>
```

Comment:
While running the tests with this PR excluded, I realized the ome-model version bump required to couple the model changes to the OMERO `metadata_files` branch was part of it. Would it be possible to move this line into a PR that we can merge separately?